charts/fybrik/templates/opa-connector/opa-connector-deployment.yaml

```diff
@@ -43,6 +43,21 @@ spec:
             - name: http
               containerPort: 8080
               protocol: TCP
+          env:
+            - name: TLS_ENABLED
+              value: {{ .Values.global.tls.enabled | quote | toString }}
+            {{- if .Values.global.tls.enabled }}
+            - name: MTLS_ENABLED
+              value: {{ .Values.global.tls.mtlsEnabled | quote | toString }}
```

Comment:
the same question about secrets separation

Reply:
I do not understand what secrets... the code above refers to mtls...